examples/secondary-care/community/repeat-dispensing/nominated-pharmacy/clinical-practitioner/single-medication-request/1-Process-Request-Dispense-200_OK.json

```diff
@@ -0,0 +1,300 @@
+{
+  "resourceType": "Bundle",
+  "id": "prescription-dispense-notification",
+  "identifier": {
+    "system": "https://tools.ietf.org/html/rfc4122",
+    "value": "a22ad970-417c-4b0d-9c18-19a54761b478"
+  },
+  "type": "message",
+  "entry": [
+    {
+      "fullUrl": "urn:uuid:BE807DAC-9DCF-45CF-91D6-70D9D58DCF34",
+      "resource": {
+        "resourceType": "MessageHeader",
+        "eventCoding": {
+          "system": "https://fhir.nhs.uk/CodeSystem/message-event",
+          "code": "dispense-notification",
+          "display": "Dispense Notification"
+        },
+        "destination": [
+          {
+            "endpoint": "https://sandbox.api.service.nhs.uk/electronic-prescriptions/$post-message",
+            "receiver": {
+              "identifier": {
+                "system": "https://fhir.nhs.uk/Id/ods-organization-code",
+                "value": "T1450"
+              },
+              "display": "NHS BUSINESS SERVICES AUTHORITY"
+            }
+          }
+        ],
+        "sender": {
+          "identifier": {
+            "system": "https://fhir.nhs.uk/Id/ods-organization-code",
+            "value": "VNCEL"
+          },
+          "display": "YORKSHIRE HOMECARE LIMITED"
+        },
+        "source": {
+          "name": "YORKSHIRE HOMECARE LIMITED",
+          "endpoint": "urn:nhs-uk:addressing:ods:VNCEL"
+        },
+        "reason": {
+          "coding": [
+            {
+              "system": "https://fhir.nhs.uk/CodeSystem/message-reason-prescription",
+              "code": "notification",
+              "display": "Notification"
+            }
+          ]
+        },
+        "response": {
+          "identifier": "999f9999-9999-9999-9ff9-f9fff9999999",
+          "code": "ok"
+        },
+        "focus": [
+          {
+            "reference": "urn:uuid:4509B70D-D8B8-EA03-1105-64557CB54A29"
+          }
+        ]
+      }
+    },
+    {
+      "fullUrl": "urn:uuid:4509B70D-D8B8-EA03-1105-64557CB54A29",
+      "resource": {
+        "resourceType": "MedicationDispense",
+        "extension": [
+          {
+            "url": "https://fhir.nhs.uk/StructureDefinition/Extension-EPS-TaskBusinessStatus",
+            "valueCoding": {
+              "system": "https://fhir.nhs.uk/CodeSystem/EPS-task-business-status",
+              "code": "0003",
+              "display": "With Dispenser - Active"
+            }
+          },
+          {
+            "url": "https://fhir.hl7.org.uk/StructureDefinition/Extension-UKCore-RepeatInformation",
+            "extension": [
+              {
+                "url": "numberOfRepeatPrescriptionsAllowed",
+                "valueUnsignedInt": 6
+              },
+              {
+                "url": "authorisationExpiryDate",
+                "valueDateTime": "2020-12-07"
+              }
```

Comment:
According to IG the first extension should be `numberOfRepeatsAllowed` and we should drop `authorisationExpiryDate` as it's not supported in MedicationDispense